2.1/build/build-rpms.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Download cc-oci-runtime.tar.gz
+curl -OkL https://github.com/01org/cc-oci-runtime/archive/2.1.0-rc.6.tar.gz
```

Comment:
Better to use params instead of hardcoded $1 (the  tag to use to build cc-oci-runtime)